src/genImage.go

```diff
@@ -0,0 +1,62 @@
+package src
+
+import (
+	"fmt"
+	"image"
+	color "image/color"
+	"image/jpeg"
+	"log"
+	"os"
+	"strconv"
+)
+
+var x = 0
+var y = 0
+var width = 400
+var height = 300
+var red uint8
+var blue uint8
+var green uint8
+
+func intToHexRGB(code uint32) (uint8, uint8, uint8) {
+	var rgb [3]uint8
+	for i := 0;i < 3;i++ {
+		rgb[i] = uint8(code >> ((3 - i) * 8) & 0xFF)
+	}
+
+	return rgb[0], rgb[1], rgb[2]
+}
+
+func generateImage(colorCode string) {
+	if len(colorCode) != 6 {
+		log.Println("Error")
```

Comment:
エラーを具体的にしましょう
```suggestion
		log.Println("generateImage: len(colorCode) != 6")
```
とか。
エラーを返して、呼び出し元で、Discordにエラーを起こしたことを通知するようにしてもいいですね。